salesforcecli/plugin-alias
# chore: migrate to Command5, Core3

### What does this PR do?

QA:
this should be tested along-side other commands that interact with the aliases, such as `org:create -a` and others

PARKING ORBIT

upgrades to core3/command5

requires https://github.com/salesforcecli/command/pull/60

mostly done as exploration for the parking-orbit process for other plugins

### What issues does this PR fix or reference?
@W-10743766@
